src/core/error.ml

```diff
@@ -17,7 +17,7 @@ and error_msg =
 	| Unify of unify_error list
 	| Custom of string
 	| Unknown_ident of string
-	| Stack of error_msg * error_msg
+	| Stack of (error_msg * Globals.pos) list
```

Comment:
Is this really easier to handle? The problem is that you can still have `Stack [Stack []]` kind of things, so you always have to manage the recursive case anyway.

Reply:
Previously this was handled with ?inner positions" printed in sub errors and stored as plain string. Now I want to keep error message and positions separate to be able to report those any way I want at the end, so I need both position and message for each stack entry.
Are you suggesting I use `Stack of (error_msg * pos) * (error_msg * pos)` and nest Stack items in there?

Reply:
Well, that would be the obvious change. But if you find the `list` approach easier to handle then I don't mind.

Reply:
Will check if that's annoying to use or not.

Reply:
Yeah I do find the list approach easier to handle, especially when dealing with conversions